src/routing/core/messaging/sending.rs

```diff
@@ -362,7 +356,12 @@ impl Core {
         variant: Variant,
         dst_pk: bls::PublicKey,
     ) -> Result<Command> {
-        let message = Message::single_src(&self.node, DstLocation::Direct, variant, None)?;
+        let message = Message::single_src(
+            &self.node,
+            DstLocation::Direct,
+            variant,
+            self.section.authority_provider().section_key,
```

Comment:
What's the message to update destion section's knowledge of us? 
i.e. what will happen when we shifted from key_A to key_B ? which means we will then send messages with key_B, which is unknown to the destination section. Then how it knows this new key_B is a trust-worth key ?